docs/development/astropy-package-template.rst

```diff
@@ -8,507 +8,13 @@ astropy-dev mailing list!
-   Generally you should aim to always have your ``master`` branch work with
-   the latest stable as well as the latest development version of astropy
-   (i.e. the astropy git master branch) and the same versions of python and
-   numpy supported by astropy. The template ``.travis.yml`` covers those
-   versions; in some circumstances you may need to limit the versions your
-   package covers.
-
-#. If you register your package with coveralls.io, then you will need
-   to modify the ``coveralls --rcfile`` line in ``.travis.yml`` file to
-   replace ``packagename`` with the name of your package.
-
-#. If you want the documentation for your project to be hosted by
-   `Read the Docs <https://readthedocs.org>`_, then you need to setup an
-   account there. The following entries in "Advanced Settings" for your
-   package on `Read the Docs <https://readthedocs.org>`_ should work:
-
-   - activate ``Install your project inside a virtualenv using setup.py install``
-   - copy these additional files from the package template into the top-level
-     directory of your package: ``.rtd-environment.yml`` and ``readthedocs.yml``.
-     Edit ``.rtd-environment.yml`` with your package name and requirements.
-   - activate ``Give the virtual environment access to the global site-packages dir.``
-
-   All other settings can stay on their default value.
-
-   If you need to mock any Python packages or C libraries that can not be
-   installed and built by Read the Docs, you should include the following mocking
-   patch before the ``Project information`` section of the ``docs/conf.py`` file::
-
-      class Mock(object):
-          def __init__(self, *args, **kwargs):
-              pass
-
-          def __call__(self, *args, **kwargs):
-              return Mock()
-
-          @classmethod
-          def __getattr__(cls, name):
-              if name in ('__file__', '__path__'):
-                  return '/dev/null'
-              elif name[0] == name[0].upper():
-                  return type(name, (), {})
-              else:
-                  return Mock()
-
-      MOCK_MODULES = ['<name of package to mock>', '<name of package to mock>']
-      for mod_name in MOCK_MODULES:
-          sys.modules[mod_name] = Mock()
-
-#. You're now ready to start doing actual work on your package.  You
-   will probably want to read over the developer guidelines of the Astropy
-   documentation, and if you are hosting your code in GitHub, you might also
-   want to read the `Github help <https://help.github.com/>`_ to ensure you know
-   how to push your code to GitHub and some recommended workflows that work for
-   the core Astropy Project.
-
-#. Good luck with your code and your science!
-
-Updating to the latest template files
-=====================================
-
-See instructions in :ref:`Item 14 above <template-changes-with-gitfu>`.
-
-
-Releasing a Python package
-**************************
+Releasing an affiliated package
```

Comment:
arrg, I should have spotted that this removes the changes made in https://github.com/astropy/astropy/pull/6797, I'll open a follow-up PR.